tests/test_openjdk_devel.py

```diff
@@ -87,13 +87,13 @@ def test_maven_present(auto_container):
     assert auto_container.connection.run_expect([0], "mvn --version")
 
 
-@pytest.mark.xfail(
-    LOCALHOST.system_info.arch == "ppc64le",
-    reason="https://bugzilla.suse.com/show_bug.cgi?id=1221983",
-)
 @pytest.mark.parametrize(
     "container,java_version",
     CONTAINER_IMAGES_WITH_VERSION,
+    marks=pytest.mark.xfail(
+        condition=LOCALHOST.system_info.arch == "ppc64le" and java_version == 11,
+        reason="https://bugzilla.suse.com/show_bug.cgi?id=1221983",
+    ),
     indirect=["container"],
 )
```

Comment:
`java_version` is not available in this scope, what you can do is roughly the following:
```suggestion
@pytest.mark.parametrize(
    "container,java_version",
    [param for param in CONTAINER_IMAGES_WITH_VERSION if param.values[1] != 11 else pytest.param(*param.values, id=param.id, marks=param.marks + pytest.mark.xfail(
        condition=LOCALHOST.system_info.arch == "ppc64le",
        reason="https://bugzilla.suse.com/show_bug.cgi?id=1221983",
    ),
    indirect=["container"],
)
```

Reply:
Thanks for the hint! Should be fixed now.